ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -11892,3 +11893,270 @@ def test_torch_tensor_quantile(
         method_flags=method_flags,
         on_device=on_device,
     )
+
+
+@st.composite
+def put_along_axis_helper(draw):
+    input_dtype, x, axis, shape = draw(
+        helpers.dtype_values_axis(
+            available_dtypes=["int64"],
+            min_num_dims=2,
+            min_dim_size=2,
+            valid_axis=True,
+            force_int_axis=True,
+            ret_shape=True,
+            min_axis=0,
+        )
+    )
+
+    if axis < 0:
+        axis = 0
+    idx_shape = list(shape)
+    idx_shape[axis] = 1
+    idx_shape = tuple(idx_shape)
+
+    idx_strategy = nph.arrays(
+        dtype=np.int64, shape=idx_shape, elements=st.integers(0, len(idx_shape) - 2)
+    )
+    indices = draw(idx_strategy)
+
+    values_strategy = nph.arrays(
+        dtype=input_dtype[0], shape=idx_shape, elements=st.integers(0, 1e3)
+    )
+    values = draw(values_strategy)
+
+    return input_dtype, x[0], indices, values, axis
+
```

Comment:
Same comments as for the paddle test. 
I think since the ivy test and all the frontend tests are so similar we could have one single helper for all of them.